xlators/mgmt/glusterd/src/glusterd-volume-set.c

```diff
@@ -1512,6 +1512,9 @@ struct volopt_map_entry glusterd_volopt_map[] = {
     {.key = "network.inode-lru-limit",
      .voltype = "protocol/server",
      .op_version = 1},
+    {.key = "network.inode-table-size",
+     .voltype = "protocol/server",
+     .op_version = GD_OP_VERSION_11_0}, /* is this right? */
```

Comment:
^^ other .op_versions here is 1, but this option will only be available from glusterfs 11.2 (or patched 11.1s).